src/main/java/FindCommand.java

```diff
@@ -0,0 +1,24 @@
+/**
+   * FindCommand inherits Command
+   * @param command the main action of the command
+   * @param task the index of task user wants to mark as done
+   * @param date is a null value for Done Command
+   */
+
+  public class FindCommand extends Command {
```

Comment:
Method is well-defined in the header but the space in between the header and method can be removed to help improve readability